packages/client/src/pages/Profile/UserDataBlock.tsx

```diff
@@ -42,27 +44,27 @@ export const UserDataBlock = () => {
         <Divider orientation="left"></Divider>
         <Row justify="space-around">
           <Col span={4}>Почта</Col>
-          <Col span={4}>{USER_DATA.email}</Col>
+          <Col span={4}>{user&& user.email}</Col>
```

Comment:
Тут и везде дальше лучше вообще не рендерить компонент, если у него нет контента
```suggestion
          {!!user && (<Col span={4}>{user.email}</Col>)}
```

Reply:
> Тут и везде дальше лучше вообще не рендерить компонент, если у него нет контента

Сделал